core/src/test/scala/unit/kafka/utils/CommandLineUtilsTest.scala

```diff
@@ -29,27 +29,44 @@ class CommandLineUtilsTest {
     CommandLineUtils.parseKeyValueArgs(argArray, false)
   }
 
+  @Test(expected = classOf[java.lang.IllegalArgumentException])
+  def testParseEmptyArgWithNoDelimiter() {
+    val argArray = Array("my.empty.property")
+    CommandLineUtils.parseKeyValueArgs(argArray, false)
```

Comment:
nit: Perhaps name the boolean arg here, `acceptMissingValue = false`? Could do the same in the existing method above as well.

Reply:
done